src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-property: 0.3s;
```

Comment:
Why did you leave only transition-property? Transition-property should only specify the property, for wich the transition will be applied (not the duration)
It also should be transition-duration, because now there's no transition at all.
There is a shorthand like `transition: color 0.3s`, and it also may be written like
```
transition-property: color;
transition-duration: 0.3s;
```

https://user-images.githubusercontent.com/66570142/152564129-1395315b-06d2-49b1-863d-dee3ae57c9c3.mov